pom.xml

```diff
@@ -2,7 +2,7 @@
          xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
     <modelVersion>4.0.0</modelVersion>
     <groupId>com.smartnews</groupId>
-    <artifactId>maven-jpa-entity-generator-plugin</artifactId>
+    <artifactId>jpa-entity-generator-maven-plugin</artifactId>
```

Comment:
Okay, it makes sense. I agree we should change the name: http://maven.apache.org/guides/plugin/guide-java-plugin-development.html

Reply:
jpa-entity-generator-maven-plugin name is better